app/zfl/ZFLLayout.tsx

```diff
@@ -1,10 +1,10 @@
 import { data, Outlet, useLocation } from "react-router";
-import type { Route } from "./+types/ZFL";
+import getFeatureFlag from "~/utils/featureFlags.server";
 import Footer from "./components/Footer";
 import PageHeader from "./components/PageHeader";
 
-export function loader({ request }: Route.LoaderArgs) {
-  if (request.url.includes("bund.de")) {
+export function loader() {
+  if (!getFeatureFlag("enable-zfl")) {
```

Comment:
I'd prefer to use constants to refer to feature flags, so that we can trace them more easily.